apps/src/applab/designProperties.jsx

```diff
@@ -226,6 +231,11 @@ export default class DesignProperties extends React.Component {
                 <DuplicateElementButton
                   handleDuplicate={this.props.onDuplicate}
                 />
+                {themesEnabled && (
```

Comment:
Should this button only be visible/enabled if there are properties on the current element that are `data-modified`?

Reply:
I decided it would better to be present all the time. Also, it is more complex to keep recomputing and adding/removing the button (e.g. as you bump a property value up and down, this button would appear and then disappear)

@ryansloan FYI

Reply:
We do have a pattern here for hiding controls that aren't applicable ("Copy to screen" disappears if you have only one screen, the Z-index controls enable/disable dynamically). I wouldn't block shipping on enabling/disabling or hiding but I think it might be useful to do this work. I opened https://codedotorg.atlassian.net/browse/STAR-554 to track, but I need to figure out which option (disable vs. hide) is right in this scenario.

Reply:
👍 We can include this in this PR